PyDSS/modes/QSTS.py

```diff
@@ -2,44 +2,37 @@
 from PyDSS.modes.abstract_solver import abstact_solver
 import math
 
-
-
-
 class QSTS(abstact_solver):
     def __init__(self, dssInstance, SimulationSettings, Logger):
         super().__init__(dssInstance, SimulationSettings, Logger)
         print("Entered QSTS mode")
         self.Settings = SimulationSettings
         self.pyLogger = Logger
-        StartDay = SimulationSettings['Project']['Start Day']
-        StartTimeMin = SimulationSettings['Project']['Start Time (min)']
-        EndTimeMin = SimulationSettings['Project']['End Time (min)']
-        sStepResolution = SimulationSettings['Project']['Step resolution (sec)']
 
-        self._Time = datetime.strptime('{} {}'.format(SimulationSettings['Project']['Start Year'],
-                                                       SimulationSettings['Project']['Start Day'] +
-                                                       SimulationSettings['Project'][
-                                                           'Date offset']
-                                                       ), '%Y %j')
-        self._Time = self._Time + timedelta(minutes=StartTimeMin)
+
+        self._Time = datetime.strptime(SimulationSettings['Project']["Start time"], "%d/%m/%Y %H:%M:%S")
```

Comment:
You used this format in the profile manager.
```
datetime.datetime.strptime(self.attrs["sTime"].decode(), '%Y-%m-%d %H:%M:%S.%f')
```
The formats should probably be consistent, at least with regard to the order and `/` vs `-`. Also, putting the day first is going to confuse Americans who typically put the month before the day. This is especially confusing because the default timestamp is `"Start time" = "1/1/2020 00:00:00"`. 

I suggest using year, month, then day. I currently have this solution in the report-metrics branch (https://github.com/NREL/PyDSS/blob/report-metrics/PyDSS/utils/utils.py#L141). As long as it isn't a performance-sensitive area it will work with several formats.

Reply:
I have added the default datetime stamp in PyDSS.common. made chages to code base so that all instance of time in string format follow the same standard